plugins/katello/developers.md

```diff
@@ -255,41 +190,25 @@ Now, lock down Katello versions to the appropriate minor, e.g. if you release `p
 mod 'katello',   '>= 0.2.0 < 0.3.0'
 ```
 
-### Step 6: Configure Koji
+### Step 7: Configure Koji
 
-Note that to perform this step, Foreman must have generated it's Koji build targets for the targeted Foreman release. This is due to how Katello tags inherit from some Foreman tags. Coordinate with the Foreman release nanny on when they plan to do that step and wait to do this step until then. Koji needs to have configuration added specific to the tags that the release will occur under. The first step is to add new build roots and tags using the following script:
+Note that to perform this step, Foreman must have generated it's Koji build targets for the targeted Foreman release. This is due to how Katello tags inherit from some Foreman tags. Coordinate with the Foreman release nanny on when they plan to do that step and wait to do this step until then. Koji needs to have configuration added specific to the tags that the release will occur under. The first step is to add new build roots and tags:
 
 ```
-https://github.com/Katello/katello-misc/blob/master/rel-eng/koji_create_buildroots.sh
+./tools koji configs/katello_XY.yaml # Make sure it looks good
+./tools koji --commit configs/katello_XY.yaml
 ```
 
-We recommend running this script in logical chunks instead of attempting to run the entire thing at once.
-
-### Step 7: Setup Mash Scripts
+### Step 8: Setup Mash Scripts
 
 In order to generate the release repositories on Koji, a mash script for the version being released needs to be added. If you do not have direct access to the koji.katello.org box, you can request access or ask someone who does to perform this step for you.
 
-Copy previous mash script to new one
-
-```
-cp /usr/local/bin/katello-mash-split.py /usr/local/bin/katello-mash-split-2.1.py
-```
-
-Open new mash script and replace the version with the new one
-
 ```
-sed -i "s#/nightly/#/2.1/#g" /usr/local/bin/katello-mash-split-2.1.py
+./tools mash-scripts config/katello_XY.yaml # Make sure it looks good
+./tools mash-scripts --commit config/katello_XY.yaml
```

Comment:
I'd add:

The generated mash scripts should be commited to tool_belt and a pull request opened. The scripts will then need to be copied onto the Koji box itself. If you do not have access, please contact someone that does or request this action on the `-dev` list.